kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/xstream/XStreamMarshaller.java

```diff
@@ -16,11 +16,11 @@
 package org.kie.server.api.marshalling.xstream;
 
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
 
 import com.thoughtworks.xstream.XStream;
+import com.thoughtworks.xstream.converters.reflection.PureJavaReflectionProvider;
```

Comment:
Sorry, I am not sure I understand...the field is final, so it needs to the value needs to be set via constructor. 

The other option would be to have a custom XStream converter. But that would also mean that we would need to keep an eye on what needs a converter and what doesn't. This seems very error prone to me.

I am open to other suggestions for sure - the `PureJavaReflectionProvider` I can think of right now.